core/src/main/resources/lib/form/file.jelly

```diff
@@ -23,29 +23,32 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form"
+         xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary">
   <st:documentation>
     Generates an input field <code>&lt;input type="file" ... /></code>
+    All unknown attributes are passed through to the field.
 
     <st:attribute name="field">
       Used for databinding.
     </st:attribute>
-    <st:attribute name="jsonAware">
-      Enable structured form submission.
+    <st:attribute name="clazz">
+      Additional CSS class(es) to add.
     </st:attribute>
-    <st:attribute name="accept">
-      Defines the file types the file input should accept. This string is a comma-separated list.
+    <st:attribute name="name"> <![CDATA[
+      This becomes @name of the <input> tag.
+      If @field is specified, this value is inferred from it.
+      ]]>
```

Comment:
```suggestion
    <st:attribute name="name">
      This becomes @name of the <input> tag.
      If @field is specified, this value is inferred from it.
```
There's no data we need to escape.

Reply:
sure was just copy pasted from textbox.jelly I think